src/app/forms/templates/general/dimensions.template.ts

```diff
@@ -18,23 +19,26 @@ export function getDimensionsSection(vehicleType: VehicleTypes, noOfAxles: numbe
             value: '',
             type: FormNodeTypes.CONTROL,
             viewType: FormNodeViewTypes.HIDDEN,
-            editType: FormNodeEditTypes.HIDDEN
+            editType: FormNodeEditTypes.HIDDEN,
+            validators: [{ name: ValidatorNames.Max, args: 99999 }]
```

Comment:
I think the `Max` validator creates the issue where it says the value must be less than 99999 which in this case the value can be 99999, `MaxLength` would be the better option. 
![image](https://user-images.githubusercontent.com/92087051/194284113-26cf81ad-4e11-43c2-8d26-cae1e14ca5ac.png)

Reply:
You are right. Thanks for pointing this out. The error messages have been corrected.

Reply:
Ah this was in my PR too